Gopkg.toml

```diff
@@ -146,13 +146,33 @@
   name = "github.com/multiformats/go-multihash"
   revision = "8be2a682ab9f254311de1375145a2f78a809b07d"
 
+[[override]]
```

Comment:
what is the reason to change all this libp2p deps?

Reply:
I couldn't build it without selecting specific versions. Now, as all dependencies are in `vendor/` I can remove them, run `dep ensure` and all looks good.

Reply:
but this is because they are pinned in Gopkg.lock. i pulled you branch reverted all changes to Gopkg.toml and Gopkg.lock. Kept following changes in Gopkg.toml:

```
diff --git a/Gopkg.toml b/Gopkg.toml
index ca65e677..308b0728 100644
--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -24,12 +24,12 @@
 
 [[constraint]]
   name = "github.com/ethereum/go-ethereum"
-  version = "=v1.8.16"
+  version = "=v1.8.17"
   source = "github.com/status-im/go-ethereum"
 
 [[constraint]]
   name = "github.com/status-im/whisper"
-  version = "=v1.2.0"
+  version = "=v1.3.0"
 
 [[override]]
   name = "github.com/golang/protobuf"
@@ -84,7 +84,7 @@
 
 [[override]]
   name = "github.com/mattn/go-isatty"
-  revision = "281032e84ae07510239465db46bf442aa44b953a"
+  revision = "3fb116b820352b7f0c281308a4d6250c22d94e27"
 
 [[override]]
   name = "github.com/pborman/uuid"
@@ -151,8 +151,8 @@
   revision = "36e9d2ebbde5e3f13ab2e25625fd453271d6522e"
 
 [[constraint]]
-  branch = "master"
   name = "github.com/status-im/rendezvous"
+  branch = "master"
 
 [[override]]
   name = **"github.com/deckarep/golang-set"**
```

executed

```
dep ensure -update github.com/status-im/rendezvous
dep ensure -update github.com/status-im/go-multiaddr-ethv4
dep ensure
```

and now i have cleaner Gopkg.lock:

```
diff --git a/Gopkg.toml b/Gopkg.toml
index ca65e677..308b0728 100644
--- a/Gopkg.toml
--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -24,12 +24,12 @@
 
 [[constraint]]
   name = "github.com/ethereum/go-ethereum"
-  version = "=v1.8.16"
+  version = "=v1.8.17"
   source = "github.com/status-im/go-ethereum"
 
 [[constraint]]
   name = "github.com/status-im/whisper"
-  version = "=v1.2.0"
+  version = "=v1.3.0"
 
 [[override]]
   name = "github.com/golang/protobuf"
@@ -84,7 +84,7 @@
 
 [[override]]
   name = "github.com/mattn/go-isatty"
-  revision = "281032e84ae07510239465db46bf442aa44b953a"
+  revision = "3fb116b820352b7f0c281308a4d6250c22d94e27"
 
 [[override]]
   name = "github.com/pborman/uuid"
@@ -151,8 +151,8 @@
   revision = "36e9d2ebbde5e3f13ab2e25625fd453271d6522e"
 
 [[constraint]]
-  branch = "master"
   name = "github.com/status-im/rendezvous"
+  branch = "master"
 
 [[override]]
   name = "github.com/deckarep/golang-set"
```

statusd compiles and can be used. maybe this is something in your cache? adding those changes to Gopkg.lock is definitely unnecessary